src/main/java/io/cdap/plugin/successfactors/source/transport/SuccessFactorsTransporter.java

```diff
@@ -45,7 +45,7 @@
  * make a rest web service call to the SAP SuccessFactors exposed services.
  */
 public class SuccessFactorsTransporter {
-  private static final String SERVICE_VERSION = "dataserviceversion";
+  public static final String SERVICE_VERSION = "dataserviceversion";
```

Comment:
why was this made public? can it just be package private?

Reply:
Changed to package private.